pkg/log/handler_iowriter_writer_file.go

```diff
@@ -0,0 +1,30 @@
+package log
+
+import (
+	"fmt"
+	"github.com/applike/gosoline/pkg/cfg"
+	"io"
+	"os"
+)
+
+type ioWriterFileSettings struct {
+	Path string `cfg:"path" default:"logs.log"`
+}
+
+func ioWriterFileFactory(config cfg.Config, configKey string) (io.Writer, error) {
+	settings := &ioWriterFileSettings{}
+	config.UnmarshalKey(configKey, settings)
+
+	return NewIoWriterFile(settings.Path)
+}
+
+func NewIoWriterFile(path string) (io.Writer, error) {
+	var err error
+	var file *os.File
+
+	if file, err = os.OpenFile(path, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0600); err != nil {
```

Comment:
What do we think about buffering the writes to a file? Does the os.File struct already take care of that?

Reply:
Don't know to be honest..  
But I don't expect this to be an issue currently..